modules/tinymce/src/core/test/ts/browser/DisabledModeTest.ts

```
@@ -0,0 +1,531 @@
+    const hook = TinyHooks.bddSetup<Editor>({
+      ...settings,
+      disabled: true,
+      setup: (ed: Editor) => {
+        ed.on('DisabledStateChange', (e) => store.add(e.state));
+      }
+    }, []);
+
+    afterEach(() => {
+      hook.editor().options.set('disabled', true);
+      store.clear();
+    });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)